scripts/calculateKpi/protocols/utils/networks.ts

```diff
@@ -14,7 +14,7 @@ export async function fetchTotalTransactionFees({
   startBlock?: number // inclusive
   endBlockExclusive?: number
 }): Promise<number> {
-  let totalTransactionFees = 0
+  let totalGasUsed = 0
```

Comment:
not sure if this should be a bigint or not. seems better safe than sorry and I can look into using bigint in a separate PR.